src/zenml/integrations/slack/alerters/slack_alerter.py

```diff
@@ -44,6 +53,8 @@ class SlackAlerterParameters(BaseAlerterStepParameters):
 
     # Set of messages that lead to disapproval in alerter.ask()
     disapprove_msg_options: Optional[List[str]] = None
+    payload: Optional[SlackAlerterPayload]
```

Comment:
I guess this needs to have a default None to be optional

```suggestion
    payload: Optional[SlackAlerterPayload] = None
```